spec/recipes/install_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../spec_helper'
+
+describe 'rundeck-server::install' do
```

Comment:
Good idea to add specs. I feel however that describing in specs the same resource than recipe does not bring much value (much effort to update versions, chef is declarative so the specs are essentially copy paste of the recipe).

Could you replace the expect statetements by `expect { chef_run }.to_not raise_error` (very simple check that ensure chef converges).